README.md

```diff
@@ -310,12 +310,26 @@ Issue](https://help.github.com/articles/creating-an-issue/).
 
 * [Microsoft Cognitive Toolkit](https://github.com/Microsoft/CNTK)
 
+* [LightGBM](https://github.com/Microsoft/LightGBM)
+
+* [Recommenders](https://github.com/Microsoft/Recommenders)
+
+* [ML.NET](https://github.com/dotnet/machinelearning)
+
+* [Infer.NET](https://github.com/dotnet/infer)
+
+* [DMTK: Microsoft Distributed Machine Learning Toolkit](https://github.com/Microsoft/DMTK)
+
+* [LightLDA](https://github.com/Microsoft/LightLDA)
+
 * [Azure Machine Learning
```

Comment:
This link needs to be changed to something new

Reply:
the azure machine learning service?  Not sure which link it should reference.  Keeping this unmodified.